openedx/core/djangoapps/credentials/tests/test_utils.py

```diff
@@ -89,3 +92,70 @@ def test_type_filter(self, mock_get_edx_api_data):
             'type': 'program',
         }
         assert kwargs['querystring'] == querystring
+
+    @override_settings(LEARNER_RECORD_MICROFRONTEND_URL=None)
+    @override_settings(CREDENTIALS_PUBLIC_SERVICE_URL="http://foo")
+    def test_get_credentials_records_url(self):
```

Comment:
There were no unit tests I could find for the existing function. I added tests for the new and old functionality.